InterventionSkillExample/skill.yaml

```diff
@@ -0,0 +1,46 @@
+camel: 1.0.0
+name: intervention-example
+title: Intervention Example
+description: Intervention Skill Example
+inputs:
+  - name: request
+    title: request
+    parameters:
+      - name: payload
+        type: object
```

Comment:
Is object the only way ? What can't the mission agent pass in 3 parameters.  Profile_id, Intevention_id. Also, how would the Skill know how the parent context is i.e the campaign, mission that is invoking it as the same Skill can be invoked by multiple interventions across missions.

Reply:
Thought of having object so we can add or remove any fields whenever needed. Will add mission name and campaign name to track the invocations.

Reply:
Fair point. Can we have a MessageType called InterventionSkillRequest , InterventionSkillResponse. This way you have a wrapper object but may include sub-elements or attributes.